src/RcppExports.cpp

```diff
@@ -5,6 +5,11 @@
 
 using namespace Rcpp;
 
+#ifdef RCPP_USE_GLOBAL_ROSTREAM
+Rcpp::Rostream<true>&  Rcpp::Rcout = Rcpp::Rcpp_cout_get();
+Rcpp::Rostream<false>& Rcpp::Rcerr = Rcpp::Rcpp_cerr_get();
+#endif
+
```

Comment:
When I check the package locally it automatically adds these lines. I'm not sure if they are strictly necessary to commit or not, though.

Reply:
this happens to me as well, have been manually deleting thus far to look at later. But I think it's fine to commit.